<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundPage</class>
 <widget class="QWidget" name="BackgroundPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="ImageList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="FileBar">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Arrangement:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="ArrangementCombo">
       <property name="minimumSize">
        <size>
         <width>192</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KSeparator" name="kseparator">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="OpenWallBtn">
       <property name="text">
        <string>Op&amp;en...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GetNewWallBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Download new wallpapers</string>
       </property>
       <property name="text">
        <string>Get New Wallpapers...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
